bbb-install-2.6.sh

```diff
@@ -937,14 +937,17 @@ install_greenlight_v3(){
   say "starting greenlight-v3..."
   docker-compose -f $GL3_DIR/docker-compose.yml up -d
   sleep 5
-  say "greenlight-v3 is ready, You can VISIT: $ROOT_URL after BBB checks complete!"
+  say "greenlight-v3 is installed and ready, You can VISIT it on: https://$HOST/ !"
+  say "To create your Greenlight administrator account kindly SEE: https://docs.bigbluebutton.org/greenlight_v3/gl3-install.html#creating-an-admin-account-1"
+
 
   if grep -q 'keycloak:' $GL3_DIR/docker-compose.yml; then
-    say "Keycloak is ready, You can VISIT: https://$HOST/keycloak after BBB checks complete!"
+    say "Keycloak is installed, You can VISIT it on: https://$HOST/keycloak !"
```

Comment:
I would change the ordering to show the password first, then give steps to complete the installation.

```
Keycloak is now installed and accessible with admin password
   https://$HOST/keycloak
.  user: admin
   password: $KCPASSWORD

To complete the configuration of Keycloak, see 
https://docs.bigbluebutton.org/greenlight_v3/gl3-external-authentication.html#configuring-keycloak
```